dir.props

```diff
@@ -189,7 +189,7 @@
   <PropertyGroup>
     <!-- Possible Values: win7 / ubuntu14.04 / osx.10.10 -->
     <RuntimeSystem Condition="'$(TargetPlatformIdentifier)' == 'Windows_NT'">win7</RuntimeSystem>
-    <RuntimeSystem Condition="'$(TargetPlatformIdentifier)' == 'Unix'">ubuntu.14.04</RuntimeSystem>
+    <RuntimeSystem Condition="'$(RuntimeSystem)' == '' and '$(TargetPlatformIdentifier)' == 'Unix'">ubuntu.14.04</RuntimeSystem>
```

Comment:
So . . . _technically_ this check isn't required, because the externally-specified global property will prevent this assignment from having any effect. Thoughts on relying on that versus leaving this change in?